global.json

```diff
@@ -1,6 +1,6 @@
 {
 	"sdk": {
-		"version": "3.1.300",
+		"version": "6.0.200",
```

Comment:
Any reason why you're targeting `6.0.200` specifically? `6.0.100` is the baseline for .NET 6, and `6.0.201` is the latest release.

Reply:
No, we can use any 6.0.x, I just used this one by muscle memory.

Reply:
`6.0.401` is now the latest release at time of writing https://dotnet.microsoft.com/en-us/download/dotnet/6.0